src/ppx/ppx_lwt.ml

```diff
@@ -133,6 +133,15 @@ let gen_top_binds vbs =
   [Vb.mk (Pat.tuple (vbs |> List.map (fun { pvb_pat; _ } -> pvb_pat)))
      [%expr Lwt_main.run [%e gen_exp vbs 0]]]
 
+let lwt_sequence mapper ~lhs ~rhs =
+  let pat = if !strict_seq then [%pat? ()] else [%pat? _] in
```

Comment:
I suggest we remove this, no need to make the new syntax be leaky :) Especially since `-no-strict-sequence` is now deprecated. Unless there is a good reason to have non-strict sequence.